org.qbittorrent.qBittorrent.yaml

```diff
@@ -2,7 +2,7 @@ app-id: org.qbittorrent.qBittorrent
 default-branch: stable
 runtime: org.kde.Platform
 sdk: org.kde.Sdk
-runtime-version: 5.15-21.08
+runtime-version: 6.4
```

Comment:
This happened in the past:
https://github.com/flathub/org.qbittorrent.qBittorrent/pull/90#issuecomment-1243379922
https://github.com/flathub/org.qbittorrent.qBittorrent/pull/90#issuecomment-1245175882
And I didn't see `6.4` is announced officially.

Reply:
From my experience RPCS3 (5.15-22.08) and OBS (6.4) pick Adwaita-dark perfectly while Tor Browser (5.15-22.08) doesn't. KeepassXC (5.15-22.08) picks it too without problems (use the 'classic' option because automatic isn't working correctly).

Reply:
The truth is KDE folks doesn't even have permission for posting under _Announcements_ in discourse forum so the link from the comment isn't best place to look at. The 5-15-22.08 was announced [here](https://discourse.flathub.org/t/new-version-of-the-kde-runtime-5-15-22-08/3710). 

The 6.x line is bit special since it's pure Qt runtime without KDE components (because KDE doesn't support Qt6 yet) so it can't be used by KDE apps. I don't know which Qt version is preferred by qbittorrent these days but if it works better with Qt6 then there is no reason to avoid it (after bit of testing). You may use Qt6 in beta branch and ask for feedback from users.

Reply:
> The truth is KDE folks doesn't even have permission for posting under _Announcements_ in discourse forum so the link from the comment isn't best place to look at. The 5-15-22.08 was announced [here](https://discourse.flathub.org/t/new-version-of-the-kde-runtime-5-15-22-08/3710).
> 
> The 6.x line is bit special since it's pure Qt runtime without KDE components (because KDE doesn't support Qt6 yet) so it can't be used by KDE apps. I don't know which Qt version is preferred by qbittorrent these days but if it works better with Qt6 then there is no reason to avoid it (after bit of testing). You may use Qt6 in beta branch and ask for feedback from users.

Oh that’s sad. So qbittorrent is preferring QT6 that’s why I would like to get the new modern library. My question is how can I get now qt6 up and running?